scipy/interpolate/tests/test_interpolate.py

```diff
@@ -906,6 +906,60 @@ def test_eval_3d(self):
         yi[:, 1, 1] = 4. * yi_
         xp_assert_close(ak(xi), yi)
 
+    def test_linear_interpolant_edge_case(self):
+        # Reference - https://github.com/scipy/scipy/issues/22011
+        x = np.array([0.0, 1.0], dtype=float)
+        y = np.array([0.55, 0.55])
+        akima = Akima1DInterpolator(x, y, axis=0, extrapolate=None)
+        xp_assert_close(akima(0.45), np.array(0.55))
+
+        x = np.array([0.0, 0.5, 1.0])
+        y = np.array([0.15, 0.67, 0.15])
+        akima = Akima1DInterpolator(x, y, axis=0, extrapolate=None)
+        xp_assert_close(akima(0.45), np.array(0.6648))
+
+        x = np.array([0.0, 1.0])
+        y = np.array([0.55,  0.55])
+        akima = Akima1DInterpolator(x, y, axis=0, extrapolate=None)
+        xp_assert_close(akima(0.45), np.array(0.55))
+
+        # 2D case
+        x = np.array([0., 1.])
+        y = np.array([0., 1.])
+        y = np.column_stack((y, 2. * y, 3. * y, 4. * y))
```

Comment:
```suggestion
        y = np.column_stack((x, 2. * x, 3. * x, 4. * x))
```

and remove `y = np.array([0., 1.0])` above

Reply:
Ah! Yes. Nice. Thanks. I don't know why I didn't see it before. Thanks a lot. :-)